### DISCUSSION TITLE
Is there any way to to detect rising of events count by 20% in 2 minutes according to previous 2 minutes interval?

### DISCUSSION THREAD

**Discussion by smi71** [SigmaHQ/sigma-specification#164] [Q&A]
I mean is it possible to describe using sigma rules?

**Comment by nasbench** [comment]
Please take a look at correlation rules https://github.com/SigmaHQ/sigma-specification/blob/main/specification/sigma-correlation-rules-specification.md

**Reply by smi71** [reply]
I'm already read it carefully but still can't figure out how to compare current counter with prevoius counter value.
